### ISSUE TITLE
CLI: Crashes when running

### ISSUE THREAD

**Issue by Hifoz** [digdir/designsystemet#2133]
### Description of the bug

The CLI crashes when trying to run it.

Tested 0.1.0-alpha.7, 0.1.0-alpha.8 and 0.1.0-alpha.10. Error first occurs in alpha.8, and is still present in alpha.10

### Steps To Reproduce

1. Run `npx @digdir/designsystemet@next` in terminal (or any version after 0.1.0-alpha.8)
2. CLI crashes

### Additional Information

Error message:
````js
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '.../node_modules/@digdir/designsystemet/dist/build/src/tokens/utils/permutateThemes' imported from .../node_modules/@digdir/designsystemet/dist/build/src/tokens/configs.js
    at new NodeError (node:internal/errors:405:5)
    at finalizeResolution (node:internal/modules/esm/resolve:324:11)
    at moduleResolve (node:internal/modules/esm/resolve:943:10)
    at defaultResolve (node:internal/modules/esm/resolve:1129:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:835:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
      code: 'ERR_MODULE_NOT_FOUND'
}
````

**Comment by mimarz** [comment]
Ah! 

Thanks for reporting this. I'll have a look what might be causing this 🤔

**Comment by mimarz** [comment]
I have published a new version ([0.1.0-alpha.11](https://www.npmjs.com/package/@digdir/designsystemet/v/0.1.0-alpha.11)) with a fix. 

Updated both tags to use this version (`latest` & `next`)
